src/main/java/core/basesyntax/AreaCalculatable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculatable {
+    public abstract double calculateArea();

```

Comment:
it's not necessary to add `public` and `abstract` to  interface methods, cause they are `public` and `abstract` by default
```suggestion
    double calculateArea();
```